.github/workflows/new_tag.yml

```diff
@@ -52,6 +54,9 @@ jobs:
   create-next-milestone:
     name: Create next milestone
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      issues: write
```

Comment:
afaict creating a milestone requires issues write (I looked this up but closed the tab -- I'm not particularly happy that the action doesn't state that explicitly).